src/script_interface/virtual_sites/VirtualSites.hpp

```diff
@@ -40,8 +40,18 @@ class VirtualSites : public AutoParameters {
                 get_value<bool>(v));            // NOLINT, bug in clang-tidy-4.0
           },
           [this]() {
-            return virtual_sites() // NOLINT, bug in clang-tidy-4.0
-                ->have_velocity(); // NOLINT, bug in clang-tidy-4.0
+            return virtual_sites()     // NOLINT, bug in clang-tidy-4.0
+                ->get_have_velocity(); // NOLINT, bug in clang-tidy-4.0
+          }},
+         {"have_quaternion",
+          [this](const Variant &v) {
+            virtual_sites()              // NOLINT, bug in clang-tidy-4.0
+                ->set_have_quaternion(   // NOLINT, bug in clang-tidy-4.0
+                    get_value<bool>(v)); // NOLINT, bug in clang-tidy-4.0
+          },
+          [this]() {
+            return virtual_sites()       // NOLINT, bug in clang-tidy-4.0
+                ->get_have_quaternion(); // NOLINT, bug in clang-tidy-4.0
           }}});
```

Comment:
Are you sure this is necessary? You just doubled the number of `NOLINT`s in Espresso. There is probably some design weakness here if Clang-Analyzer trips over this code so much.

Reply:
Maybe there is a design weakness, I don't know who put the NOLINTS there before me, I just adapted.

Reply:
That is a known false-positive in clang-tidy (4.0), which is fixed in later releases. As far as I can tell the code is correct, as it is not actually evaluated in the constructor (this would be UB), but merely used to define a lambda.